scripts/ralph/acceptance-prd209.sh

```
@@ -0,0 +1,84 @@
+# Acceptance gate — PRD-209 Open-Core Phase 0 (fresh-clone boot + honest CI).
+set -uo pipefail
+cd "$(dirname "$0")/../.." || exit 1
+FAIL=0
+check() {
+  local name="$1"; shift
+  echo ""; echo "── $name"
+  if bash -c "$1"; then echo "   ✅ PASS: $name"; else echo "   ❌ FAIL: $name"; FAIL=1; fi
+}
+
+check "orchestrator-full-suite (no NEW failures vs documented env baseline)" \
+  'OUT=$(cd orchestrator && python3 -m pytest --timeout=90 --timeout-method=thread -o faulthandler_timeout=120 -p no:cacheprovider -q 2>&1 | tail -200); echo "$OUT" | grep -qE "[0-9]{4} passed" && ! echo "$OUT" | grep "^FAILED" | grep -vE "prd172|dr_restore|composio" | grep -q .'
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)